### PR TITLE
feat: update disabled button and outline button styles

### PR DESCRIPTION
**Pull request by ajlu**
## Description

💁‍  Update disabled styles for buttons
https://app.clubhouse.io/qantasaccommodation/story/75051/roo-ui-disabled-button

## Related issues

💪

## Looks like this

Screen Shot 2021-08-19 at 2 41 49 pm
Screen Shot 2021-08-19 at 2 42 05 pm

🤨

---

**PR check-list**

🚨 **IMPORTANT:** Please review the [guidelines for contributing](./CONTRIBUTING.md) to this repository first! 👀

- [ ] You've requested a review from at least ONE person. Preferably a roo-ui maintainer ([Angus](https://github.com/angusfretwell), [Philip](https://github.com/philipwindeyer), [Eric](https://github.com/eneo5541), [Mike](https://github.com/memcc), or [Long](https://github.com/KieraDOG)).
  - Note: If your PR has not been reviewed within two days of requesting a review, feel free to seek reviews from other [team members of Qantas Hotels](https://github.com/orgs/hooroo/people).
- [ ] Your final merge commit message is in the format `<type>(<scope>): <description>` as per [the contributing guide](https://github.com/hooroo/roo-ui/blob/master/.github/CONTRIBUTING.md#commit-naming)
  - Example: `fix(dependencies): minor version bump to resolve security vulnerability in set-value`

### CODE REVIEW

**Comment by ehyland** [comment]
👷 Deploy Preview for *roo-ui* processing.


🔨 Explore the source changes: e3740d9db4b2ce69e83bf60d75de75a4d1d175fc

🔍 Inspect the deploy log: [https://app.netlify.com/sites/roo-ui/deploys/611de6c6d33df60008da5a65](https://app.netlify.com/sites/roo-ui/deploys/611de6c6d33df60008da5a65?utm_source=github&utm_campaign=bot_dl)

**Review by chienleng — APPROVED** [review]

File: src/components/Button/__snapshots__/Button.test.js.snap [hunk 1]
```diff
@@ -6,6 +6,12 @@ exports[`<Button /> block renders correctly 1`] = `
 </ContextConsumer>
 `;
 
+exports[`<Button /> disabled renders correctly 1`] = `
+<ContextConsumer>
+  <Component />
+</ContextConsumer>
+`;
```
Comment (mrm007) [inline src/components/Button/__snapshots__/Button.test.js.snap:13]: Not your fault, but these are useless snapshots. If you use `mountWithTheme` the styles will show up in the snapshot, which is much better than `<Component />`.

File: src/components/Button/Button.test.js [hunk 2]
```diff
@@ -51,4 +51,14 @@ describe('<Button />', () => {
...
+
+  describe('disabled', () => {
+    beforeEach(() => {
+      wrapper = shallowWithTheme(<Button disabled>Hello world</Button>, theme);
+    });
+
+    it('renders correctly', () => {
+      expect(wrapper).toMatchSnapshot();
+    });
+  });
```
Comment (mrm007) [inline src/components/Button/Button.test.js:63]: I understand this is the pattern, but I think there's an opportunity to do a bit of clean up.
This pattern wastes a lot of space and can be rewritten as:

```suggestion
  test('disabled renders correctly', () => {
    wrapper = shallowWithTheme(<Button disabled>Hello world</Button>, theme);
    expect(wrapper).toMatchSnapshot();
  });
```

I also recommend using `mountWithTheme` so the styles are captured in the snapshot, as per my other comment.

File: src/components/Button/__snapshots__/Button.test.js.snap (hunk 1 above)
Comment (ajlu) [reply src/components/Button/__snapshots__/Button.test.js.snap]: I agree, refactored

File: src/components/Button/Button.test.js (hunk 2 above)
Comment (ajlu) [reply src/components/Button/Button.test.js]: updated, also changed wrapper to a constant which I think is easier to read.

File: src/components/OutlineButton/README.md [hunk 3]
```diff
@@ -18,11 +18,12 @@ export default <OutlineButton>Hello world</OutlineButton>;
...
-| Name       | Description          | Type     | Default  | Required? |
-| :--------- | :------------------- | :------- | :------- | :-------- |
-| `variant`  | use a theme variant  | `string` | 'default | -         |
-| `rounded`  | use a rounded border | `bool`   | false    | -         |
-| `disabled` | disable the button   | `bool`   | false    | -         |
+| Name       | Description                | Type     | Default  | Required? |
+| :--------- | :------------------------- | :------- | :------- | :-------- |
+| `variant`  | use a theme variant        | `string` | 'default | -         |
+| `rounded`  | use a rounded border       | `bool`   | false    | -         |
+| `block`    | render a full width button | `bool`   | false    | -         |
```
Comment (mrm007) [inline src/components/OutlineButton/README.md:25]: 👍 👍

**Review by mrm007 — APPROVED** [review]
Awesome work! 🌮

**Pull request merged**